sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeFileClient.cs

```diff
@@ -2819,12 +2795,11 @@ Response<Blobs.Models.BlobDownloadStreamingResult> response
         /// A <see cref="RequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
-        // TODO #27253
-        //[EditorBrowsable(EditorBrowsableState.Never)]
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public virtual Response<FileDownloadInfo> Read(
-            HttpRange range = default,
-            DataLakeRequestConditions conditions = default,
-            bool rangeGetContentHash = default,
+            HttpRange range,
+            DataLakeRequestConditions conditions,
+            bool rangeGetContentHash,
             CancellationToken cancellationToken = default)
```

Comment:
Remove `= default` and add the suppression to this overload.  Here's an example - https://github.com/Azure/azure-sdk-for-net/blob/2fa1dc608acf4b051c175b672b5363eaa668870f/sdk/storage/Azure.Storage.Blobs/src/AppendBlobClient.cs#L1002